db/seed.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+ (echo "BEGIN;"; \
+ cat ./db/seeds/development/reset.sql; \
+ cat ./db/seeds/development/users.sql; \
+ cat ./db/seeds/development/categories.sql; \
+ cat ./db/seeds/development/forums.sql; \
+ cat ./db/seeds/development/topics.sql; \
+ cat ./db/seeds/development/posts.sql; \
+ echo "COMMIT;") \
+ | psql --host $PGHOST $PGDATABASE -U $PGUSER 
```

Comment:
clever! You may want to pull those files out of /db so Massive doesn't bother loading them though.

Reply:
kk

i wish sqitch had a seeds option along with deploy and revert to make life easier. so i should move all sqitch related files out of the db folder?

Reply:
I kind of see run-once-for-test-data stuff as outside Massive's purview -- you don't want the possibility of running `db.seeds.development.categories()` during runtime. So that script shouldn't be in the runtime Massive instance's /db directory. You could spin up an instance with those scripts attached to initialize if you liked but that's overkill, your `cat` solution here works great.

Reply:
ahh i see. i can also configure massive so it doesn't use /db as the scripts directory right?

Reply:
yeah you can customize that [in the loader configuration](https://dmfay.github.io/massive-js/connecting.html#loader-configuration-and-filtering).